cross/java-11-openjdk/Makefile

```diff
@@ -45,11 +50,11 @@ CONFIGURE_ARGS += --with-version-opt=LTS
 CONFIGURE_ARGS += --with-debug-level=release
 CONFIGURE_ARGS += --with-native-debug-symbols=none
 CONFIGURE_ARGS += --enable-unlimited-crypto
-CONFIGURE_ARGS += --with-zlib=bundled
-CONFIGURE_ARGS += --with-libjpeg=bundled
-CONFIGURE_ARGS += --with-giflib=bundled
-CONFIGURE_ARGS += --with-libpng=bundled
-CONFIGURE_ARGS += --with-lcms=bundled
+CONFIGURE_ARGS += --with-zlib=system
+CONFIGURE_ARGS += --with-libjpeg=system
+CONFIGURE_ARGS += --with-giflib=system
+CONFIGURE_ARGS += --with-libpng=system
+CONFIGURE_ARGS += --with-lcms=system
```

Comment:
I doubt that this will work as expected.
As there is no option like `--with-zlib=$(INSTALL_DIR)/$(INSTALL_PREFIX)` it will take the library installed on the host system, and this will definitly lead to errors like
 _zlib returned Z_VERSION_ERROR: compile time and runtime zlib implementations differ_

Reply:
Again. yes and no. If you want to be 100% safe I can add the paths to the cross libs.

Reply:
From my understanding, `--with-zlib` is only valid with `bundled`or `system` but not with a path.

Reply:
correct.

I can add $(INSTALL_DIR)/$(INSTALL_PREFIX)/lib to

CONFIGURE_ARGS += --with-extra-cflags="$(CFLAGS) -I$(INSTALL_DIR)/$(INSTALL_PREFIX)/include"
CONFIGURE_ARGS += --with-extra-cxxflags="$(CPPFLAGS) -I$(INSTALL_DIR)/$(INSTALL_PREFIX)/include"
CONFIGURE_ARGS += --with-extra-ldflags="$(LDFLAGS) -L$(INSTALL_DIR)/$(INSTALL_PREFIX)/lib"

and hope for the best.

But. as of now it works fine the package i compiled with the docker image.

Reply:
ah. not needed

spksrc.tc-flags.mk:LDFLAGS += -L$(INSTALL_DIR)/$(INSTALL_PREFIX)/lib

Thats why it works ;) ;)

Reply:
@hgy59 unless you want to do more I think the PR is ready for a merge?